app/components/PredictionMarkets/PredictionMarketDetailsTable.jsx

```diff
@@ -0,0 +1,151 @@
+                align: "left",
+                onCell,
+                render: dataItem => {
+                    return <span>{dataItem.fee}</span>;
+                }
+            },
+            {
+                key: "actions",
+                title: counterpart.translate("prediction.overview.action"),
+                align: "left",
+                onCell,
+                render: dataItem => {
+                    return (
+                        <div
+                            style={{
+                                display: "flex",
+                                flexDirection: "column",
+                                alignItems: "center"
+                            }}
+                        >
+                            {this.props.currentAccountId &&
+                            dataItem.opinionator ===
+                                this.props.currentAccountId ? (
+                                <Button
+                                    onClick={() => {
+                                        this.props.onCancel(dataItem);
+                                    }}
+                                >
+                                    {counterpart.translate(
+                                        "prediction.details.cancel"
+                                    )}
+                                </Button>
+                            ) : (
+                                <Button
+                                    onClick={() => {
+                                        this.props.onOppose(dataItem);
+                                    }}
+                                >
+                                    {counterpart.translate(
+                                        "prediction.details.oppose"
+                                    )}
+                                </Button>
+                            )}
+                        </div>
+                    );
+                }
+            }
+        ];
+    }
+
+    render() {
+        let pagination = {
+            hideOnSinglePage: true,
+            pageSize: 20,
+            showTotal: total =>
+                counterpart.translate("utility.total_x_items", {
+                    count: total
+                })
+        };
+
+        let filteredOpinions = this.props.marketData.opinions.filter(item => {
+            let accountName = ChainStore.getAccount(item.opinionator)
+                ? ChainStore.getAccount(item.opinionator).get("name")
+                : null;
+            return (
+                (accountName + "\0" + item.opinion)
+                    .toUpperCase()
+                    .indexOf(this.props.detailsSearchTerm) !== -1
+            );
+        });
+
+        let i = 0;
+        filteredOpinions = filteredOpinions.map(item => ({
+            ...item,
+            key: `${item.order_id}${i++}`
+        }));
+
+        return (
+            <div style={{paddingTop: "50px"}} key="overview-table">
+                <Table
```

Comment:
use the paginatedlist component to streamline UX

Reply:
Done